domains/transfer/Transfer.API/API.IntegrationTests/Cvr/Api/v2024_01_03/Controllers/CvrMultipleResponseTests.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using API.Cvr.Api.v2024_01_03.Dto.Responses;
+using API.IntegrationTests.Factories;
+using FluentAssertions;
+using VerifyTests;
+using VerifyXunit;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+using Xunit;
+
+namespace API.IntegrationTests.Cvr.Api.v2024_01_03.Controllers;
+//This should be in the same class as CvrControllerTests, but WireMock does not Co-operate with that.
+[UsesVerify]
+public class CvrMultipleResponseTests : IClassFixture<TransferAgreementsApiWebApplicationFactory>
+{
+    private readonly TransferAgreementsApiWebApplicationFactory factory;
+    private readonly WireMockServer server;
+
+    public CvrMultipleResponseTests(TransferAgreementsApiWebApplicationFactory factory)
+    {
+        this.factory = factory;
+        server = WireMockServer.Start();
+        factory.CvrBaseUrl = server.Url!;
+    }
+
+    [Fact]
+    public async Task GetCvrCompany_WhenWrongPlusRight_ShouldReturnRightOnly()
+    {
+        var wrongPlusRightCvr = new List<string> { "123", "28980671", "39315041" };
+        server.ResetMappings();
+        server
+            .Given(Request.Create().WithPath("/cvr-permanent/virksomhed/_search").UsingPost())
+            .RespondWith(Response.Create()
+                .WithStatusCode(200)
+                .WithBodyFromFile("Cvr/Api/v2024_01_03/Controllers/CvrControllerTests.cvr_response2.json")
```

Comment:
What is response2?

Reply:
a json document containing an elasticsearch response body, with 2 companies.

Reply:
Can you rename it to be self explaning? eg. cvr_multiple_companies.json

Reply:
Yes:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/5a4183c995c2ea621c69e2c1983c53e884cb8d17